app/views/reports/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2>Reports</h2>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Format</th>
+      <th>Size</th>
```

Comment:
I am removing the size. It was not required, since it is variable UK and XI have different MB.